### ISSUE TITLE
Line styles in Snail mode

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#4463]
> I’ve been working [...] on a task and he’s suggested that in Debrief I plot tracks with different line types to show different phases. I’ve tried to change line type from solid to dotted when a phase changes (using “@” for solid and “A” for dotted) and viewing this in snail mode works fine, but when viewing in normal mode if the last line type specified is dotted then all the lines for every phase are shown as dotted. Is there a way around this?

I'm pretty sure it's a shortcoming in the Snail rendering. We should investigate.

**Comment by saulhidalgoaular** [comment]
I understand that client wants to create one track, having different line styles depending on the phase (Similar to what Debrief does now with colors).

Debrief was initially created to have only 1 line style per track. Last line style is applied to the whole track, and it applies to normal and snail mode.

https://github.com/debrief/debrief/blob/88e808d521f5d44dc3353eac7a88ce8d964ac1df/org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Replay/ImportReplay.java#L2195
 
We could change this behavior if needed.

**Comment by helenayele** [comment]
Yes, in D-Lite, Line styles are not applied on the track, but works in full debrief. Please compare the two products with the same property value:
[![Image from Gyazo](https://i.gyazo.com/477087a937d4286895d0174a82ec05ad.png)](https://gyazo.com/477087a937d4286895d0174a82ec05ad)

**Comment by ayeshare** [comment]
@helenayele @IanMayo Is this related to #4360 ?

**Comment by helenayele** [comment]
hmm the original issue is related to only full debrief, right?

**Comment by helenayele** [comment]
@ayeshare  , it is not related to #4360. the issue exists in both products.
For example I construct a rep file, by modifiying the last 9 points of boat1.rep file to have dot-dash(different line style) from the others points like this.
............
951212 112500.000 NELSON   @C   22  7  2.11 N 21 44 58.28 W 236.1   2.0      0 
951212 112600.000 NELSON   @C   22  6 58.62 N 21 45  5.84 W 287.7   2.0      0 
951212 112700.000 NELSON   @C   22  7  0.63 N 21 45 14.91 W 334.9   2.0      0 
951212 112800.000 NELSON   @C   22  7  7.00 N 21 45 19.26 W 340.7   2.0      0 
951212 112900.000 NELSON   @C   22  7 14.33 N 21 45 23.01 W 346.1   2.0      0 
951212 113000.000 NELSON   @C   22  7 21.35 N 21 45 25.55 W  21.4   2.0      0 
951212 113100.000 NELSON   @C   22  7 26.42 N 21 45 22.64 W  21.0   2.0      0 
951212 113200.000 NELSON   @C   22  7 30.93 N 21 45 20.10 W  19.3   2.0      0 
951212 113300.000 NELSON   @AB00  22  7 35.20 N 21 45 17.91 W  20.3   2.0      0 
951212 113400.000 NELSON   @AB00   22  7 39.70 N 21 45 15.48 W  20.6   2.0      0 
951212 113500.000 NELSON   @AB00   22  7 45.19 N 21 45 12.46 W  20.1   2.0      0 
951212 113600.000 NELSON   @AB00   22  7 51.59 N 21 45  9.04 W  20.3   2.0      0 
951212 113700.000 NELSON   @AB00   22  7 58.91 N 21 45  5.08 W  20.8   2.0      0 
951212 113800.000 NELSON   @AB00   22  8  6.64 N 21 45  0.78 W  20.2   2.0      0 
951212 113900.000 NELSON   @AB00   22  8 14.70 N 21 44 56.44 W  20.1   2.0      0 
951212 114000.000 NELSON   @AB00   22  8 22.95 N 21 44 52.03 W  20.2   2.0      0 
951212 114100.000 NELSON   @AB00   22  8 30.26 N 21 44 48.10 W  20.2   2.0      0
And saved this. 
And when I open it in both full debrief and D-Lite, the whole tracks line styles is turned to Dot-dash in Normal display mode. But it should be applied only to the selected 9 points only.
[![Image from Gyazo](https://i.gyazo.com/2669c287a2db9945d6aa45f32cb96237.png)](https://gyazo.com/2669c287a2db9945d6aa45f32cb96237)

But in Snail mode , the dot-dash line style is only applied to the selected positions only.
[![Image from Gyazo](https://i.gyazo.com/1e122b5be3b6ab0cac0a5cdc226fdfd2.png)](https://gyazo.com/1e122b5be3b6ab0cac0a5cdc226fdfd2)

**Comment by IanMayo** [comment]
Nice work @helenayele , great to have you back 🤸‍♂️
